build.sbt

```diff
@@ -21,42 +12,42 @@ val Versions = new {
   val snakeYaml = "1.33"
   val snakeYamlEngine = "2.5"
   val previousCirceYamls = Set("0.14.0", "0.14.1", "0.14.2")
+
+  val scala212 = "2.12.17"
+  val scala213 = "2.13.10"
+  val scala3 = "3.2.1"
+
+  val scalaVersions = Seq(scala212, scala213, scala3)
 }
 
 val docMappingsApiDir = settingKey[String]("Subdirectory in site target directory for API docs")
 
-ThisBuild / crossScalaVersions := Seq("2.12.15", "2.13.8", "3.2.0")
+ThisBuild / scalaVersion := Versions.scala213
+ThisBuild / crossScalaVersions := Versions.scalaVersions
 
-val root = project
-  .in(file("."))
-//  .settings(commonSettings)
+val root = tlCrossRootProject.aggregate(
+  `circe-yaml-common`,
+  `circe-yaml`,
+  `circe-yaml-v12`
+)
+
+lazy val `circe-yaml-common` = crossProject(JVMPlatform)
+  .crossType(CrossType.Pure)
+  .withoutSuffixFor(JVMPlatform)
```

Comment:
Why make everything a `crossProject`? This project will never cross-compile for JS (assuming it sticks with the snake yaml parser).

Reply:
Following changes in circe-fs2.  Wasn't sure if it was needed.  Undone now.